index.html

```diff
@@ -11744,30 +11788,37 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 					</tr>
 					<tr>
 						<th class="value-name" scope="row">true</th>
+						<th class="state-description" scope="row">True</th>
 						<td class="value-description">Indicates the popup is a <rref>menu</rref>.</td>
 					</tr>
 					<tr>
 						<th class="value-name" scope="row">menu</th>
+						<th class="state-description" scope="row">Menu</th>
 						<td class="value-description">Indicates the popup is a <rref>menu</rref>.</td>
 					</tr>
 					<tr>
 						<th class="value-name" scope="row">listbox</th>
+						<th class="state-description" scope="row">Listbox</th>
 						<td class="value-description">Indicates the popup is a <rref>listbox</rref>.</td>
 					</tr>
 					<tr>
 						<th class="value-name" scope="row">tree</th>
+						<th class="state-description" scope="row">Tree</th>
 						<td class="value-description">Indicates the popup is a <rref>tree</rref>.</td>
 					</tr>
 					<tr>
 						<th class="value-name" scope="row">grid</th>
+						<th class="state-description" scope="row">Grid</th>
 						<td class="value-description">Indicates the popup is a <rref>grid</rref>.</td>
 					</tr>
 					<tr>
 						<th class="value-name" scope="row">dialog</th>
+						<th class="state-description" scope="row">Dialog</th>
 						<td class="value-description">Indicates the popup is a <rref>dialog</rref>.</td>
 					</tr>
 				</tbody>
 			</table>
+			<p>The attribute's <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a> is the Undefined state, and its <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> is the Menu state.</p>
```

Comment:
Again, here, the missing value default being the `Undefined` state means `.ariaHasPopup` will return `"undefined"` which I don't think is desirable. Instead:

```suggestion
			<p>The attribute has no <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a>, and its <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> is the Menu state.</p>
```